parser/openfoodfacts_taxonomy_parser/test.txt

```diff
@@ -0,0 +1,37 @@
+# test taxonomy
+
+stopwords:fr: aux,au,de,le,du,la,a,et
+
+synonyms:en:passion fruit, passionfruit
+
+synonyms:fr:fruit de la passion, fruits de la passion, maracuja, passion
```

Comment:
@BryanH01 Use the [updated test taxonomy](https://github.com/openfoodfacts/taxonomy-editor/blob/main/backend/sample/test.txt) from main.